### DISCUSSION TITLE
Aside is on the left margin on left page but on the right margin the right page

### DISCUSSION THREAD

**Discussion by yoonghm** [quarto-dev/quarto-cli#6559] [Q&A] [answered]
### Description

I am looking for available features in Quarto allowing me to create mirror margin: aside is on the left margin on left page but on the right margin the right page

![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/6d6610c5-870a-4753-85ea-f281daeb002c)

https://github.com/quarto-dev/quarto-gallery/blob/main/page-layout/tufte.qmd put aside on the right margin.

**Comment by mcanouil** [comment, marked as answer]
I am guessing you are talking about LaTeX/PDF format, see below an example.

````qmd
---
title: "Quarto Playground"
format:
  pdf:
    documentclass: article
    classoption: twoside
---

This is a playground for Quarto.

## Page 1

{{< lipsum 2 >}}

::: {.aside}
{{< lipsum 1 >}}
:::

{{< pagebreak >}}

## Page 2
  
{{< lipsum 2 >}}

::: {.aside}
{{< lipsum 1 >}}
:::

````

**Comment by yoonghm** [comment]
It works. Thanks!
